templates/hp_procurve_show_interfaces_brief.textfsm

```diff
@@ -0,0 +1,29 @@
+Value PORT (\S+)
+Value TYPE (\S+|\s+)
```

Comment:
All white and non-white space is the same as `*`, can you change or better yet see if there is another strategy to do this with?